### DISCUSSION TITLE
How to add new datasets to delta fuseki?

### DISCUSSION THREAD

**Discussion by tejbloke** [afs/rdf-delta#115] [Q&A] [answered]
Hi,

I'm looking at rdf-delta as high availability alternative to fuseki, i tried a "Simple setup" and found that jena fuseki administrative apis such as /$/datasets, /$/server, /$/ping are not accessible in delta-fuseki instances. As /$/datasets endpoint is not available, what options do i have to add new datasets and sync them across delta-fuseki instances in a rdf-delta setup.

Thanks,
Teja

**Comment by afs** [comment, marked as answer]
The form of Fuseki used here is the "[Fuseki main](https://jena.apache.org/documentation/fuseki2/fuseki-main.html)" version with no UI or administration modules. Configuration is by [configuration file](https://afs.github.io/rdf-delta/delta-fuseki-config.html).

RDF Delta replicates changes between servers but does not replicate configuration. There is dynamic reconfiguration of the cluster while running.

If you have a configuration file, you can copy it to each machine being setup.
